<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KecepatanGravitasi</class>
 <widget class="QDialog" name="KecepatanGravitasi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>377</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="posisiLintang">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>260</y>
     <width>142</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>150</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Hitung</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>290</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Kecepatan Gravitasi :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>260</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Posisi Lintang             :</string>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>340</y>
     <width>166</width>
     <height>25</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>41</y>
     <width>176</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Masukan Tinggi Terbang : </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="tinggiTerbangHasil">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>230</y>
     <width>142</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Tinggi Terbang           : </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="tinggiTerbang">
   <property name="geometry">
    <rect>
     <x>201</x>
     <y>41</y>
     <width>142</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="kecGravitasi">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>290</y>
     <width>142</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="sudutLintang">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>80</y>
     <width>142</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>18</x>
     <y>80</y>
     <width>176</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Masukan Sudut Lintang   : </string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>tinggiTerbang</tabstop>
  <tabstop>sudutLintang</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>tinggiTerbangHasil</tabstop>
  <tabstop>posisiLintang</tabstop>
  <tabstop>kecGravitasi</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
